api_app/script_analyzers/observable_analyzers/phishtank.py

```diff
@@ -0,0 +1,37 @@
+import requests
+import logging
+from api_app.exceptions import AnalyzerRunException
+from api_app.script_analyzers.classes import ObservableAnalyzer
+from intel_owl import secrets
+import base64
+
+logger = logging.getLogger(__name__)
+
+
+class Phishtank(ObservableAnalyzer):
+    def set_config(self, additional_config_params):
+        self.api_key_name = additional_config_params.get(
+            "api_key_name", "PHISTANK_API_KEY"
+        )
+
+    def run(self):
+        result = {}
+        data = {
+            "url": base64.b64encode(self.observable_name.encode("utf-8")),
+            "format": "json",
+        }
+        session = requests.session()
```

Comment:
still not needed

Reply:
Hey, yes. I tried the above. The issue also exists when we use no API key. In this scenario, header would be an empty dictionary.

Reply:
Could you please give us proof about your statement?

Reply:
Please see my comment below which explains this behavior.